### PR TITLE
doc: update document for "--acpidev_pt" and "--mmiodev_pt" features

### PR DESCRIPTION
**Pull request by lifeix**
Add guild how to add ACPI device and MMIO device passthrough support through
the cmdline.

Signed-off-by: Li Fei1 <fei1.li@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/user-guides/acrn-dm-parameters.rst [hunk 1]
```diff
@@ -276,6 +276,30 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+   * - :kbd:`--acpidev_pt <HID>`
+     - This option is to enable ACPI device passthrough support. The HID is a
+       mandatory parameter for this option which is the Hardware ID of the ACPI
+       device.
+
+       Example::
+
+          --acpidev_pt MSFT0101
+
+       To pass through a TPM (which HID is MSFT0101) ACPI device to a UOS.
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst:288]:
```suggestion
       To pass through a TPM (which HID is MSFT0101) ACPI device to a User VM.
```

File: doc/user-guides/acrn-dm-parameters.rst [hunk 2]
```diff
@@ -276,6 +276,30 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+
+   * - :kbd:`--mmiodev_pt <MMIO_Regions>`
+     - This option is to enable MMIO device passthrough support. The MMIO_Regions
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst:291]:
```suggestion
     - This option is to enable MMIO device passthrough support. The ``MMIO_Regions``
```

File: doc/user-guides/acrn-dm-parameters.rst [hunk 3]
```diff
@@ -276,6 +276,30 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+       is a mandatory parameter for this option which are the MMIO resources of the
+       MMIO device.
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst:293]: Based on the example below, it sounds like that parameter needs to be the base address followed by the length of the region, both separated by a comma. Is that correct? If true, can you explain that here?

Can one include multiple regions using that parameter? If so, what separator should be used between two regions?

File: doc/user-guides/acrn-dm-parameters.rst [hunk 4]
```diff
@@ -276,6 +276,30 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+
+       Example::
+
+          --mmiodev_pt 0xFED40000,0x00005000
+
+       To pass through a MMIO device to a UOS. The MMIO device has a MMIO region.
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst:299]:
```suggestion
       To pass through a MMIO device to a User VM. The MMIO device has a MMIO region.
```

File: doc/user-guides/acrn-dm-parameters.rst (hunk 3 above)
Comment (lifeix) [reply doc/user-guides/acrn-dm-parameters.rst:293]: Hi Geoffroy 
Thanks for your comments. Yes, will explain this. Besides, now we only support one MMIO region. Will change the MMIO_regions to MMIO_region. Is this OK. (Will detail how we separate MMIO regions when we support it.)

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**